sites/footers/old-footers-fr.html

```diff
@@ -7,12 +7,87 @@
 	],
 	"altLangPage": "old-footers-en.html",
 	"secondlevel": false,
-	"dateModified": "2025-04-30",
+	"dateModified": "2025-06-12",
 	"share": "true"
 }
 ---
 <div class="wb-prettify all-pre hide"></div>
 
+<h2>Version 8.1 (demoted)</h2>
+<ul>
+	<li>Ajout d'un nouveau lien "Gérer les événements de la vie"</li>
+</ul>
+<p>Exemple pratique: <a href="demoted/footers-v8.1-fr.html">Pied de page version 8.1</a></p>
+<details>
+	<summary>Code final attendu</summary>
+	<pre><code>&lt;footer id="wb-info">
+		&lt;h2 class="wb-inv">À propos de ce site&lt;/h2>
```

Comment:
Fix indentation of the code sample. Everything is one tab too far, starting at line 24.